### ISSUE TITLE
Auto-run during acceptance tests

### ISSUE THREAD

**Issue by trentmwillis** [trentmwillis/ember-a11y-testing#3]
The current API for this is very explicit, in that you have to explicitly call a function anytime you wish to test for accessibility. Accessibility testing should be more implicit, as we should expect it to happen. Therefore, we should schedule these tests to run automatically during acceptance tests (maybe even unit tests?) after each rendering update to the DOM.

I imagine, exploration of performance improvements will be needed to make this not super slow.

**Comment by nathanhammond** [comment]
Issue was relevant to the older version of ember-a11y-testing.
